dist/common/scripts/scylla_io_setup

```diff
@@ -142,6 +142,39 @@ if __name__ == "__main__":
             yaml.dump({ "disks": [ disk_properties ] }, properties_file,  default_flow_style=False)
             ioconf = open(etcdir() + "/scylla.d/io.conf", "w")
             ioconf.write("SEASTAR_IO=\"--io-properties-file={}\"\n".format(properties_file.name))
+        elif gcp_instance().is_gce_instance() and (is_debian_variant() or is_redhat_variant()):
+            idata = gcp_instance()
 
+            if idata.is_recommended_instance():
+                disk_properties = {}
+                disk_properties["mountpoint"] = datadir()
+                nr_disks = idata.nvmeDiskCount
+                if nr_disks >= 1 & nr_disks < 4:
+                    disk_properties["read_iops"] = 179861 * nr_disks
+                    disk_properties["read_bandwidth"] = 737304512 * nr_disks
+                    disk_properties["write_iops"] = 99963 * nr_disks
+                    disk_properties["write_bandwidth"] = 409628288 * nr_disks
+                elif nr_disks >= 4 & nr_disks <= 8:
+                    disk_properties["read_iops"] = 721719
+                    disk_properties["read_bandwidth"] = 2951769344
```

Comment:
How come the rates for 4 disks are the same as the rates for 8 disks?

Reply:
1 controller is used, they have 3TB nvme cards, which they partition in 8x375GB
they limit the speed for lower than 4 cards (they share same nvme card between smaller instances)
for more than 4 you get the full card without sharing

Reply:
Is this documented somewhere? please add a link.

Is there a risk that we'll lose some of the bandwidth if another VM on the same host uses the remaining disks?

Reply:
documented on https://cloud.google.com/compute/docs/disks/#localssds (I will add this to code as doc pointer), resp. in detail on https://cloud.google.com/compute/docs/disks/local-ssd#performance
+ if you try to add a local nvme in https://console.cloud.google.com/compute/instancesAdd?project=upload-179716 it will show you changes to performance for 1-4 and then 6-8 will stay same as per above doc (the testing confirms this behaviour)

Documentation doesn't show what happens if more VMs share the same card, above docs suggest however what I mentioned before (that >4 disks == you get the whole controller for yourself).

Reply:
It's really strange. from the 16 and 24 numbers, I expect 100k read IOPS per disk. But in the 1-4 range, you get 170k read IOPS per disk. in the 4-8 range, it stops increasing.

Reply:
I'd say it's totally in line with table https://cloud.google.com/compute/docs/disks/local-ssd#nvme
my empirical results are even more optimistic than their table.

Reply:
It's better to use their table then. There could be variance between machines.

Reply:
adjusted per their table, note that for 16 and 24 disks our results were LOWER than theirs (on recommended VM sizes (see https://docs.google.com/spreadsheets/d/1lMsGTfX0HQURH1L-A8czQXHhwLARQHBjXUDNWoEwrl4/edit#gid=2122362307 ), so I used our results instead
I will retest though with Pavels fixes in seastar iotune, for now keeping what was measured for 16 and 24 disk bandwidths